modules/3d-tiles/src/lib/parsers/helpers/parse-3d-tile-gltf-view.js

```diff
@@ -29,6 +29,12 @@ export function parse3DTileGLTFViewSync(tile, arrayBuffer, byteOffset) {
     throw new Error('glTF byte length must be greater than 0.');
   }
 
+  // Save gltf up axis
+  tile.gltfUpAxis =
```

Comment:
OK. 

Note that I am not sure about storing this on the `cesium-ion` options. I think the people using gltfUpAxis in the deck.gl repo were not using the cesium ion service.

This could be just `3d-tiles` option?

Reply:
Fix it. Now fetch it from 3d-tiles options